### ISSUE TITLE
The given credential is rejected by the attribute condition

### ISSUE THREAD

**Issue by sumanbikram** [terraform-google-modules/terraform-example-foundation#1362]
### TL;DR

Rugging the default github action created by TF but it throing error. Any idea? Also
[run](https://github.com/sumanbikram/Bootstrap/actions/runs/12392988487/job/34593311030#step:3:20)
google-github-actions/auth failed with: failed to generate Google Cloud federated token for //iam.googleapis.com/***: {"error":"unauthorized_client","error_description":"The given credential is rejected by the attribute condition."}

<img width="562" alt="image" src="https://github.com/user-attachments/assets/343df0bc-022f-4b7d-a0ef-76b454084ff5" />

### Terraform Configuration

```hcl
not applicable
```

### Terraform Version

```sh
latest
```
